criticality_score/generate.py

```diff
@@ -143,9 +143,11 @@ def main():
             try:
                 repo = run.get_repository(repo_url)
                 if not repo:
-                    logger.error(f'Repo not found: {repo_url}')
+                    logger.error(f'Repo is not found: {repo_url}')
                     break
                 output = run.get_repository_stats(repo)
+                if not output:
+                    logger.error(f'Repo is empty: {repo_url}')
```

Comment:
This logger.error repeated in 2 places can just be moved inside get_repository_stats
if not repo.last_commit:
  logger.error(f'Repo is empty: {repo_url}')
  return None